ydb/core/tx/schemeshard/ut_column_build/ut_column_build.cpp

```diff
@@ -215,7 +216,7 @@ Y_UNIT_TEST_SUITE(ColumnBuildTest) {
 
         Ydb::TypedValue defaultValue;
         defaultValue.mutable_type()->set_type_id(Ydb::Type::UINT64);
-        defaultValue.mutable_value()->set_text_value("1111");
+        defaultValue.mutable_value()->set_uint64_value(1111); // TODO: check invalid value
```

Comment:
It looks like it needs to be validated somewhere here: https://github.com/ydb-platform/ydb/blob/main/ydb/core/tx/schemeshard/schemeshard_build_index__create.cpp#L161

Reply:
okay from my side here. I'll check this unit a bit later.